homework_12/myproject/homework_model/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+from django.db.models import TextField
+
+# Create your models here.
+
+
+class Person(models.Model):
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+
+    def __str__(self) -> str:
+        return f"{self.first_name} {self.last_name}"
+
+
+class Student(Person):
+    db_table = "Student"
+
+
+class Teacher(Person):
+    db_table = "Teacher"
+
+
+class Homework(models.Model):
+    db_table = "Homework"
+    text = models.TextField()
+    deadline = models.DateTimeField()
+    created = models.DateTimeField()
+    teacher = models.ForeignKey(Teacher, on_delete=models.CASCADE)
+
+    def __str__(self) -> TextField:
```

Comment:
actually, 'models.TextField()' on 25 line is a descriptor call, so this method returns string value. if you are interested, you can read this article https://medium.com/@yuchen_71541/how-django-uses-python-descriptors-e0077caae152